h5p_proplist.go

```diff
@@ -40,6 +44,26 @@ func (p *PropList) Close() error {
 	return p.closeWith(h5pclose)
 }
 
+// Sets the size of the chunks used to store a chunked layout dataset.
+// https://support.hdfgroup.org/HDF5/doc/RM/RM_H5P.html#Property-SetChunk
+func (p *PropList) SetChunk(ndims uint, dim []uint) error {
+	var c_dim *C.hsize_t
+	c_dim = (*C.hsize_t)(unsafe.Pointer(&dim[0]))
```

Comment:
couldn't we directly initialize `c_dim`? (_ie:_ merge line 50 and 51 into just one line)
also: we should probably properly handle 0-size and `nil` `dim` slices.

Reply:
Fixed